ui/page/dcrdex/market.go

```diff
@@ -1606,34 +1561,93 @@ func (pg *DEXMarketPage) HandleUserInteractions() {
 			pg.lotsOrAmountEditor.ExtraText = pg.selectedMarketOrderBook.baseSymbol
 		}
 	}
+}
+
+// HandleUserInteractions is called just before Layout() to determine if any
+// user interaction recently occurred on the page and may be used to update the
+// page's UI components shortly before they are displayed.
+// Part of the load.Page interface.
+func (pg *DEXMarketPage) HandleUserInteractions(gtx C) {
+	if pg.isDEXReset() {
+		return
+	}
+
+	dexc := pg.AssetsManager.DexClient()
+	if pg.serverSelector.Changed(gtx) {
+		selectedServer := pg.serverSelector.Selected()
+		xc, err := dexc.Exchange(selectedServer)
+		if err != nil && xc.Auth.EffectiveTier == 0 /* need to post bond now */ {
+			pg.ParentNavigator().ClearStackAndDisplay(NewDEXOnboarding(pg.Load, selectedServer))
+		} else {
+			pg.lastSelectedDEXServer = selectedServer
+			pg.setServerMarkets()
+		}
+	}
+
+	if pg.addServerBtn.Clicked(gtx) {
+		pg.ParentNavigator().ClearStackAndDisplay(NewDEXOnboarding(pg.Load, ""))
+	}
+
+	if pg.openOrdersBtn.Clicked(gtx) {
+		pg.orders = nil // clear orders
+		pg.openOrdersDisplayed = true
+		go pg.refreshOrders()
+	}
+
+	if pg.marketSelector.Changed(gtx) {
+		pg.fetchOrderBook()
+	}
+
+	if pg.orderHistoryBtn.Clicked(gtx) {
+		pg.orders = nil // clear orders
+		pg.openOrdersDisplayed = false
+		go pg.refreshOrders()
+	}
+
+	if pg.seeFullOrderBookBtn.Clicked(gtx) {
+		// TODO: display full order book
+		log.Info("Display full order book")
```

Comment:
log an error like this " button click listener for full order book view is not implemented"

Reply:
hm.. it's old code, not in this PR's scope, but I will fix it